package.json

```diff
@@ -30,6 +30,7 @@
     "is-integer": "^1.0.7",
     "is-number": "^7.0.0",
     "json-schema-ref-parser": "^9.0.1",
+    "jsonpath-plus": "^4.0.0",
```

Comment:
Check if the other 2 json libs offer this functionality

Reply:
One is for JSON Validation, and the other is for JSON Data Parser which will be removed as part of cleanup milestone.
So, we can have this for querying and fetching a part of json data using jsonpath.